doc/source/examples/workflow.rst

```diff
@@ -0,0 +1,25 @@
+ComPWA Workflow
+===============
+
+In the following pages, we show how to use ComPWA via the Python interface.
+We illustrate this workflow with the decay
+:math:`J/\psi \rightarrow \gamma \pi^0 \pi^0`, using the helicity formalism.
+
+Usually, you go through the following four steps to perform your analysis:
+
+.. toctree::
+    :maxdepth: 2
+    :glob:
+
+    workflow/*
+
+:doc:`Let’s go! <workflow/1_create_model>`
+
+.. note::
+
+    Each of these steps can be downloaded as Jupyter notebooks
+    `here <https://github.com/ComPWA/pycompwa/tree/master/examples>`__.
```

Comment:
This link will also not work if you moved the examples dir.

Reply:
Actually this note has become redundant with the [added download buttons](https://github.com/ComPWA/pycompwa/pull/74/commits/274e16c9dd8b324977ac74556dd383337c0221bf).

Reply:
Ok, I hope those buttons will work if the notebooks are copied to the docs

Reply:
Yes, it makes use of the [Sphinx `download` role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-download)

Reply:
Ok, I thought those links are auto generated. Ok nvm